operations/bills.md

```diff
@@ -107,7 +107,15 @@ Returns all bills, optionally filtered by customers, identifiers and other filte
                 "Value": {
                     "ItalianFiscalCode": null
                 }
-            }
+            },
+            "EnterpriseData": {
+                AdditionalTaxIdentifier: "XY00112233445",
+                CompanyName: "The Sample Hotel Group AS",
+                BankAccount: "CZ3808000000000012345678",
+                BankName: "CESKA SPORITELNA A.S.",
+                Iban: "CZ6508000000192000145399",
+                Bic: "GIBACZPX"
+            }            
```

Comment:
@KaliCZ Why is this done like this? It's a part of the array which means, for 100 bills, we serialize this 100x. How come this does not have approval from anyone from connectivity? cc @vitezp

Reply:
I thought that Approval from Mike is good enough.

Well it can actually be different for each bill, that's why it's serialized like this. It's a snapshot in time when the bill was closed.

Reply:
@vitasamek I thought that Approval from Mike is good enough.

Well it can actually be different for each bill, that's why it's serialized like this. It's a snapshot in time when the bill was closed.

Reply:
@KaliCZ approval from @MikeAdamsMews is not good at all. He is not responsible for the contract. He is responsible for the content and structure of the documentation. Not for the contract of the API. That one is fully owned by the connectivity TECH team.